keyboards/stront/config.h

```diff
@@ -0,0 +1,95 @@
+// Copyright 2023 zzeneg (@zzeneg)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_PIN_SWAP
+#define SERIAL_USART_TX_PIN GP0
+#define SERIAL_USART_RX_PIN GP1
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 500U
+
+/* any side can be master by default, enable split sync to support it */
+#define EE_HANDS
+#define SPLIT_LED_STATE_ENABLE
+#define SPLIT_MODS_ENABLE
+#define SPLIT_POINTING_ENABLE
+
+/* SPI config for display/touchpad */
+#define SPI_DRIVER SPID1
+#define SPI_SCK_PIN GP10
+#define SPI_MOSI_PIN GP11
+#define SPI_MISO_PIN GP12
+
+/* touchpad config */
+#define POINTING_DEVICE_RIGHT
+#define POINTING_DEVICE_CS_PIN GP9
+#define POINTING_DEVICE_ROTATION_180
+#define CIRQUE_PINNACLE_DIAMETER_MM 40
+
+/* LCD config */
+#define LCD_DC_PIN GP12
```

Comment:
This is doubled up with MISO.  Is that intentional?

Reply:
Yes - same pin is used as DC for LCD on one half and as MISO on another. I could use `SPI_MISO_PIN` in `qp_st7789_make_spi_device` instead but feel like it looks worse.